include/npcomp/Dialect/Torch/IR/TorchOps.td

```diff
@@ -15,8 +15,31 @@ class Torch_Op<string mnemonic, list<OpTrait> traits = []>
     : Op<Torch_Dialect, mnemonic, traits> {
 }
 
-def Torch_DummyOp : Torch_Op<"dummy"> {
-  let summary = "Dummy placeholder op until more is defined";
+def Torch_KernelCall : Torch_Op<"kernel_call"> {
+  let summary = "Calls a Torch custom kernel";
+  let description = [{
+    Torch kernel calls are matched by the runtime based on signature, including
+    the fully qualified kernel name (i.e. "namespace::name") and the tuple of
+    argument types.
+
+    This op models such an invocation. Note that depending on the layer that
+    the invocation is done in Torch, arguments may be reversed (i.e. the boxed
```

Comment:
The phrasing here is awkward. I interpreted "arguments may be reversed" as applying to this op, not the boxed stack. 

Maybe something like:

```
    Torch kernel calls are matched by the runtime based on signature, including
    the fully qualified kernel name (i.e. "namespace::name") and the tuple of
    argument types. This op models such an invocation.

  Caveat: When interacting with the PyTorch boxed interpreter stack, the arguments follow a convention that the top of stack is the first argument.
```